agent/src/main/scala/trackedfuture/runtime/TrackedAwait.scala

```diff
@@ -0,0 +1,43 @@
+package trackedfuture.runtime
+
+import java.util.concurrent.TimeoutException
+
+import trackedfuture.util.ThreadLocalIterator
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Awaitable}
+
+/**
```

Comment:
Default IDE comments do not contain useful information.  (better create AUTHORS section in README)